### PR TITLE
test(ignores) icrease coverage

### PR DESCRIPTION
**Pull request by jmlavoier**
#### What
- Add tests to increase `lib/ignores function.
- Add default to options prop so that it doesn't break when options isn't undefined. 

#### Related Issue
https://github.com/coderaiser/putout/issues/12

### CODE REVIEW

**Comment by jmlavoier** [comment]
@coderaiser I tried to understand the lint problem, but I didn't. This problem is happening in a place where I haven't changed. I don't know why, could catch that? If you explain me how I can fix it, I do.

**Comment by jmlavoier** [comment]
I've found the problem, that's alright now. If you have some complain or advice, don't hesitate to tell me. Like you have explained me before, It's important to well understand the project and contribute better each PR.

File: packages/putout/lib/ignores.spec.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    
+    t.equal(result, expected);
+    t.end();
+});
+
+test('putout: ignores: ignore true', (t) => {
+    const result = ignores('x/y', 'x/y/z/*.js', {ignore: ['z/*.js']});
+    
+    const expected = true;
+
```
Comment (coderaiser) [inline packages/putout/lib/ignores.spec.js]: Please remove a blank line between declarations on lines 49 and 40. Everything else is OK :).

**Comment by jmlavoier** [comment]
I was wondering in another test case. There is an different exception when the relative path is in another path in parent directory.

**Comment by coderaiser** [comment]
Perfect, thank you :)

**Pull request merged**